library/package.json

```diff
@@ -14,7 +14,7 @@
   "author": "TensorWorks Pty Ltd",
   "license": "MIT",
   "dependencies": {
-    "@epicgames-ps/lib-pixelstreamingfrontend-ue5.2": "^0.5.0",
+    "@epicgames-ps/lib-pixelstreamingfrontend-ue5.2": "^0.5.1",
```

Comment:
Double check this release tag as I see:

https://github.com/EpicGames/PixelStreamingInfrastructure/releases/tag/UE5.2-0.6.1